src/cleanfigure.m

```diff
@@ -206,8 +203,9 @@ function pruneOutsideBox(meta, handle)
   %hasMarkers = ~strcmp(marker,'none');
   hasLines = true;
```

Comment:
BTW, i always stumble upon those lines. Can't we just remove those lines and the checks altogether?